memory-viz/src/automate.ts

```diff
@@ -254,15 +254,15 @@ function separateObjects(objects) {
 
     for (const item of objects) {
         if (
-            item.type === "BLANK" &&
+            item.type === ".blank" &&
             (item.width === undefined || item.height === undefined)
         ) {
             console.log(
                 "WARNING :: An object with name='BLANK' exists with missing dimension information " +
                     "(either the width or the height is missing). This object will be omitted in the memory model" +
                     " diagram."
             );
-        } else if (item.stack_frame) {
+        } else if (item.type.includes("frame")) {
```

Comment:
This is too broad; we may eventually add new types that contain the string "frame". Better here to explicitly define a list of "frame types" and check inclusion in that list.